code/zFulpstationCode/fulp_overwrite_vars.dm

```diff
@@ -254,10 +254,51 @@
 //** Engiborgs now start with a manipulator for wall mounted frames and basic
 //** electronics which can be upgraded to hold stock parts and circuitboards
 //*****************************************************************************
-
+/obj/item/robot_module/standard
+	basic_modules = list(
+		/obj/item/laser_pointer, //replaced flash
```

Comment:
friendly reminder that the current niche of standard cyborgs is being an alternative loadout for secborgs (that trades your disabler and stun baton for the ability to self-heal and do (more) things other than validhunt), and the loss of their flashes would absolutely cripple them in that role (as they then don't really have anything that they can combo into their cablecuffs (aside from harm intent borgspinning someone into a wall, but that is both harmful and requires buckling someone to yourself, which is incredibly difficult/nigh-impossible to do if they're not willing or hardstunned (or braindead))). without their flashes (and/or their cuffs), standard cyborgs are basically just worse engiborgs (you lose your RCD, access to the engiborg upgrades, polycircuit whatsits, and half of your tools (INCLUDING YOUR SCREWDRIVER), in exchange for... a health analyzer and an epinephrine injector?).

Reply:
I am fine leaving standards with flashes 

I am trying to understand why you think engineering/medical/janitor cyborgs should be the peacekeepers of the station. I know that a flash helps people stop fighting, but why do you think a cyborg's objective should include stopping that? Do you think that adds value or fun to the game?

Reply:
why do heads of staff get a flash, then?

Reply:
Personally I think part of a head of staff's duty is to maintain order in their department, you should be able to go a department's head or a member of security if something is going wrong, and they should be able to solve that problem with you. 
This PR should probably infer that I don't think anyone that isn't a security cyborg (or standard cyborg, I think that is fine) should be included in this peace keeper role.

Also worth mentioning, if we were HRP and on a whitelisted server, I would be OK with removing heads batons and flashes.

Reply:
and in my opinion, you should be able to run to your local silicon and beg them for help as well.

also, under, say, asimov, a silicon is obligated to attempt to stop a nonhuman if they're harming a human. if you take away one of their main ways to stop said nonhuman nonlethally, what do you think they're going to try to do instead?

Reply:
Sounds like a disagreement over expectation then. I do not like cyborgs acting as security when they are not security modules, but you do like that. 
Cyborgs have plenty in their kit to stop human harm excluding their flashes.